internal/distro/fedora30/distro_test.go

```diff
@@ -1,11 +1,38 @@
 package fedora30_test
 
 import (
-	"github.com/osbuild/osbuild-composer/internal/distro/fedora30"
 	"reflect"
 	"testing"
+
+	"github.com/osbuild/osbuild-composer/internal/distro/fedora30"
+	"github.com/osbuild/osbuild-composer/internal/osbuild"
+	"github.com/stretchr/testify/assert"
 )
 
+type arch struct {
```

Comment:
You should not need to redefine the internal structs to test the public functions. Otherwise I think the tests look good.